### ISSUE TITLE
Proposal: add flag to validator to output lines with errors to file

### ISSUE THREAD

**Issue by jochemb** [cBioPortal/cbioportal#838]
Running the validator on a file with a large number of errors outputs only the first 3 lines now.  To improve functionality I suggest to have an optional flag that writes all lines with errors to a file.

This could be done in the following format:

```
>filename1.txt | error_message1
line1, line2, line3, line4, line5, line6
>filename1.txt | error_message2
line3, line7, line8, line9
>filename2.txt | error_message1
line1
```

**Comment by jochemb** [comment]
@fedde-s Could you have a look please

**Comment by fedde-s** [comment]
I implemented this in commit thehyve/cbioportal@672fd6beb35b45ad883bf4f8ce9e86d540afa072, although I see you suggested commas and spaces between line numbers and I used newlines. Do you prefer commas?

Example output from the version I implemented:
```
>brca_tcga_pub.maf | Normal sample id not in list of sample ids configured in corresponding metafile. Please check your metafile field 'normal_samples_list'.
7
8
9
10
11
12
13
14
15
```

**Comment by fedde-s** [comment]
Done in thehyve/cbioportal@5e440921afe7ff2a93b19ca92882f80d92447ec5, click the link to see the change.
